### PR TITLE
danese-covid19-understanding.yml 

### PR DESCRIPTION
**Pull request by albedan**
Thanks for contributing your submission to the e-Rum2020 CovidR contest.

**For CovidR contributors:**

- [x] The title of this PR reflects your contribution
- [x] The pull request only includes one new `.yml` file under `contest/contributions`
- [x] Make sure "Allow edits from maintainers" is checked
- [x] Send an email to covidrcontest@erum.io (for further direct communication down the road)
     - Subject: "Submission to CovidR: PR #00", where #00 is the number of this Pull Request
     - Content: A full link to this Pull Request
     - Optionally include your social media profiles (Twitter, LinkedIn, Facebook): we will use them to `@mention` you while sharing the contribution through our channels.

---

For CovidR maintainers:

- [x] Label the PR as "submission"
- [x] Check the validity of the submission
- [x] Assess completeness of metadata and possibly adjust the `.yml` file-name
- [x] Check the integration of the new page in the website gallery (via Actions' artifact or locally)
- [x] Make sure the email to covidrcontest@erum.io was received
- [x] Add a field with the current `date` and approx. merge time (e.g. `2020-04-01-16:30` for April 1st, 4:30 PM)
- [x] Squash-merge the pull request (commit headline will be OK, consider cleaning-up the details)
- [x] The initial comment should be automatically created, otherwise comment manually with the raw content of [vote-comment.md](../tree/master/contest/_tools/vote-comment.md) and an initial like
- [x] Provide here the link to the contribution as published to the gallery: https://milano-r.github.io/erum2020-covidr-contest/danese-covid19-understanding.html
- [x] Instruct contributors on how to include their badge from the gallery page, share on social media, and get a [free ticket](https://www.eventbrite.it/e/covidr-erum-2020-pre-conference-event-tickets-104133026686).
- [x] Consolidate e-mailed information.

### CODE REVIEW

File: contest/contributions/danese-covid19-understanding.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+title: Understanding COVID-19 through Italian excess deaths analysis
```
Comment (riccardoporreca) [inline contest/contributions/danese-covid19-understanding.yml]: Title-case
```suggestion
title: Understanding COVID-19 through Italian Excess Deaths Analysis
```

File: contest/contributions/danese-covid19-understanding.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+author: Alberto Danese
+abstract: |
+  Why are we expecting to understand COVID-19, when most of the analysis rely on the same biased, unreliable and dull data? Death and cases are counted differently across countries. In Italy, regions and provinces have different capabilities and policies when it comes to testing the population. All this makes drawing conclusions a hard and risky task. 
+  Here I suggest an original approach based on overall deaths data from ISTAT, available from 2015 to 2020, done 100% in R.
+  I focus on excess deaths with respect to previous years and answer many questions with a robust quantitative approach. Are men really more hit by COVID-19 than women? Is it true that only the elderly are affected? What really happened in Bergamo and other Italian provinces?
+menu_entry: Understanding COVID-19 through Italian excess deaths analysis
```
Comment (riccardoporreca) [inline contest/contributions/danese-covid19-understanding.yml]: Shorter `menu_entry`
```suggestion
menu_entry: Understanding COVID-19 with Excess Deaths
```

File: contest/contributions/danese-covid19-understanding.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+repository: https://github.com/albedan/covidr-excess-deaths-analysis
+type: webpage
+content: https://bookdown.org/connect/#/apps/4119
```
Comment (riccardoporreca) [inline contest/contributions/danese-covid19-understanding.yml]: We cannot embed the specified `content` as a gallery page via an `iframe`, see the minimal embedding example 
[danese-covid19-understanding.zip](https://github.com/Milano-R/erum2020-covidr-contest/files/4603302/danese-covid19-understanding.zip)
```
Load denied by X-Frame-Options: “deny” from “https://bookdown.org/connect/#/apps/4119”, site does not permit any framing. Attempted to load into “file:///home/riccardo/Downloads/danese-covid19-understanding.html”.
```
This is probably prevented by the RSConnect wrapper which itself embeds the actual content: I could retrieve the ultimate content as https://bookdown.org/content/4119
```suggestion
content: https://bookdown.org/content/4119
```

**Review by riccardoporreca — CHANGES_REQUESTED** [review]
@albedan, Thanks for your PR. I am proposing a few minor cosmetic changes and a rather technical one to allow embedding in the CovidR Gallery.
Let me now you you have questions, otherwise feel free ti go ahead and "Commit suggestion".

File: contest/contributions/danese-covid19-understanding.yml (hunk 2 above)
Comment (albedan) [reply contest/contributions/danese-covid19-understanding.yml]: How about "Understanding COVID-19 with Excess Deaths"?
I'd like to point out that the analysis is about understanding what's actually going on, it's applied to Italian data but could be applied elsewhere ;)

Comment (riccardoporreca) [reply contest/contributions/danese-covid19-understanding.yml]: Thanks @albedan, I am going to finalize the submission with your proposal.

File: contest/contributions/danese-covid19-understanding.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+title: Understanding COVID-19 through Italian Excess Deaths Analysis
+author: Alberto Danese
+abstract: |
+  Why are we expecting to understand COVID-19, when most of the analysis rely on the same biased, unreliable and dull data? Death and cases are counted differently across countries. In Italy, regions and provinces have different capabilities and policies when it comes to testing the population. All this makes drawing conclusions a hard and risky task. 
+  Here I suggest an original approach based on overall deaths data from ISTAT, available from 2015 to 2020, done 100% in R.
+  I focus on excess deaths with respect to previous years and answer many questions with a robust quantitative approach. Are men really more hit by COVID-19 than women? Is it true that only the elderly are affected? What really happened in Bergamo and other Italian provinces?
+menu_entry: Understanding COVID-19 with Excess Deaths
+repository: https://github.com/albedan/covidr-excess-deaths-analysis
+type: webpage
+content: https://bookdown.org/content/4119
```
Comment (riccardoporreca) [inline contest/contributions/danese-covid19-understanding.yml:10]:
```suggestion
content: https://bookdown.org/content/4119
date: 2020-05-09-11:45
```

**Review by riccardoporreca — APPROVED** [review]

File: contest/contributions/danese-covid19-understanding.yml [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+date: 2020-05-09-11:45
```
Comment (riccardoporreca) [inline contest/contributions/danese-covid19-understanding.yml]:
```suggestion
date: 2020-05-09-23:45
```

**Review by riccardoporreca — APPROVED** [review]

**Comment by riccardoporreca** [comment]
@albedan, your submission is now accepted and visible at https://milano-r.github.io/erum2020-covidr-contest/danese-covid19-understanding.html :tada: 
You can click on the **eRum2020::Covidr badge** in the page to obtain the badge code you can embed in your repository.

Furthermore, feel free to spread your contribution on social media with the link above, using hashtags `#erum2020` `#covidr`, and invite the community to like the contribution using the corresponding button.

Last but not least: You can get your **free ticket for the pre-conference event** at https://www.eventbrite.it/e/covidr-erum-2020-pre-conference-event-tickets-104133026686

**Pull request merged**